spectacles/validators/content.py

```diff
@@ -22,6 +32,31 @@ def _get_personal_folders(self) -> List[int]:
                 personal_folders.append(folder["id"])
         return personal_folders
 
+    def _get_all_subfolders(self, input_folders: List[int]) -> List[int]:
+        result = []
+        all_folders = self.client.all_folders(self.project.name)
```

Comment:
Looks like we still call this twice. Are you okay with that?

Reply:
I think I am. It's now once for personal folders and once for subfolders, which I think I'm happy with. Would you be inclined to consolidate further?

Reply:
Only if it's straightforward to do so. I don't think 1 extra call is going to have much of a performance hit.